### DISCUSSION TITLE
升级了最新版的docker后，无法访问api了

### DISCUSSION THREAD

**Discussion by nikissXI** [linweiyuan/go-chatgpt-api#156] [Q&A] [answered]
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/31379266/6fde1bb2-6327-46ad-be30-ba1bb67293f5)
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/31379266/69826cf6-33c9-4b28-ae6b-1bde47200c81)

**Comment by linweiyuan** [comment, marked as answer]
还没起来，提示 is ready 才可以访问

本地网络代理 55556？

go-chatgpt-api 是59999？

**Reply by nikissXI** [reply]
噢，貌似是ip被block还是代理写错了，改着改着好了XD
